hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/monitor/invariants/TestMetricsInvariantChecker.java

```diff
@@ -30,8 +30,11 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Timeout;
 
+import static org.junit.Assume.assumeNotNull;
```

Comment:
If we find that the relevant classes already use org.junit.jupiter, we can use the JUnit 5 syntax.

Reply:
Thank you.
Fixed.